tests/tests_integration/test_api/test_datapoints.py

```diff
@@ -754,6 +754,34 @@ def test_iterate_multiple_requests(self, cognite_client, all_test_time_series, r
         else:
             pytest.fail("Too few iterations/datapoints returned")
 
+    @pytest.mark.parametrize("retrieve_arrays", (False, True))
+    def test_iterating_object_aggregates(self, cognite_client, all_test_time_series, retrieve_arrays):
+        # Bug prior to 7.74.2, when iterating object aggregate datapoints like `min_datapoint` or `max_datapoint`,
+        # a 'ValueError: Unsupported aggregate' would be raised because `include_status` did not have a default value.
```

Comment:
a quick question, if user had supplied value for include_status, iteration would work?

Reply:
actually one more, on API side parameter looks optional, so what was failing?

Reply:
> a quick question, if user had supplied value for include_status, iteration would work?

Yes.

> actually one more, on API side parameter looks optional, so what was failing?

The problem was SDK code relating to the new object aggregates. The API happily returned datapoints, but unpacking failed.